developer/configuration.md

```diff
@@ -4,7 +4,7 @@ Reaction can be configured on startup with a combination of environment variable
 
 Reaction uses `/private/settings/reaction.json` for the configuration of Reaction packages and [Meteor.settings](http://docs.meteor.com/#/full/meteor_settings) for initial administrator and server setup.
 
-## Environment
+## Environmental variables
```

Comment:
Is "Environmental variable" the way we refer to these? I don't see us doing that elsewhere but I could be wrong.